pages/[id]/bridge.tsx

```diff
@@ -219,6 +220,81 @@ const Bridge = () => {
   const isAmountError = errorType === ErrorType.ERC20AmountError;
   const isEthError = errorType === ErrorType.InsufficientNativeCurrencyError;
 
+  const connectButton = (
+    <div className="mt-5">
```

Comment:
all these elements have `mt-5` as a class. is it possible to remove from each, and add to the classnames on ln 389?

Reply:
Good point, updated here: [568fabf](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/153/commits/568fabff3c0ddcc83187e75be1f73db728be1208)